payments/swish/index.md

```diff
@@ -18,223 +18,86 @@ sidebar:
+## Purchase Flow
 
-Swish payments from a mobile device made either through an app or via a
-mobile browser on the mobile device that hosts the Swish app.
-The flow redirects the payment dialogue to Swedbank Pay Payments,
-that will handle the required user dialogue.
+If you are using the browser based flow, the consumer enters his or hers phone
+on Swedbank Pay's payment page (redirect) or in the `iframe` (seamless view)
+after the payment is created.
 
-### Swish e-commerce, Redirect to Payments
+![screenshot of the Swish redirect payment page with number input][swish-payment]{:height="500px" width="425px"}
 
-_Available in ![swedish flag][se-image]{:height="15px" width="15px"}_
+After pushing the pay button, a push message is sent to the phone, asking the
+consumer to confirm the purchase in the Swish app. If you are using the redirect
+option, the consumer will be redirected back to the merchant's site.
 
-Swish payments initiated by the consumer in a browser in equipment other than
-the mobile device that hosts the Swish app.
-The flow redirects the payment dialogue to Swedbank Pay Payments,
-that will handle the required user dialogue / mobile number input.
+In the mobile purchase flow, the payment page or `iframe` will only have a pay
+button, and no number input. The Swish app will be launched automatically when
+you push the pay button.
 
-### Swish m-commerce, Direct API integration
+## Good To Know
 
-_Available in ![swedish flag][se-image]{:height="15px" width="15px"}_
+### Payment Type
 
-Swish payments from a mobile device made either through an app or via a
-mobile browser on the same mobile device.
+Swish is one of the instruments using one-phase payments. The `sale` is done
+when the consumer successfully confirms in the app, capturing the funds
+instantly. The `abort` operation is still available, but the `cancel` and
+`capture` operations are not. The `reversal`, if needed, is done by the
+merchant at a later time. Read more about the [different
+operations][after-payment] and the [payment resource][payment-resource].
 
-### Swish e-commerce, Direct API integration
+### Certificate
 
-_Available in ![swedish flag][se-image]{:height="15px" width="15px"}_
-
-Swish payments initiated by the consumer in a browser in equipment other
-than the mobile device that hosts the Swish app.
-
-### Payment Link
-
-_Available in ![swedish flag][se-image]{:height="15px" width="15px"}_
-
-Generate a Payment Link that can be sent to the consumer via e-mail or SMS,
-so the consumer may pay at a later moment.
-Payment links can be implemented for all payment instruments supporting Redirect
-to hosted payment page
+As a part of the setup, you will need a Swish certificate. As a Swedbank
+costumer you can apply for Swish as part of [Swedbank Pay Settlement
+Service][settlement-service]), and utilize the Swedbank Pay Technical Supplier
+Certificate. A [Swedbank Pay sales representative][payex-mailto] can assist you
```

Comment:
Should we use `@swedbankpay.*` e-mail addresses here, perhaps? Complicated that we need to mention all country specific variations, but better than using PayEx addresses, I think?